src/apyfloatarray_wrapper.cc

```diff
@@ -477,6 +477,252 @@ void bind_float_array(nb::module_& m)
             )pbdoc"
         )
 
+        .def(
+            "squeeze",
+            &APyFloatArray::squeeze,
+            nb::arg("axis") = nb::none(),
+            R"pbdoc(
+            Removes dimensions of size 1 at the specified axes, if no axís is given removoves all dimensions with size 1.
+
+            Parameters
+            ----------
+            axis : tuple, int, optional
+                The axes to squeeze, a given axis with a size other than 1 will result in an error. no given axes  will be remove all dimensions of size 1.
+
+            Returns
+            -------
+            :class:`APyFloatArray`
+
+            Raises
+            -------
+            ValueError
+                If given an axis of a size other than 1 a ValueError will be thrown.
+            IndexError
+                If a specified axis is outside of the existing number of dimensions for the array
+
+            )pbdoc"
+        )
+
+        .def(
+            "sum",
+            &APyFloatArray::sum,
+            nb::arg("axis") = nb::none(),
+            R"pbdoc(
+            Returns the cumulative sum of the elements along specified axes.
+
+            Parameters
+            ----------
+            axis : tuple, int, optional
+                The axes to summate across. Will summate the whole array if no int or tuple is specified
+
+            Returns
+            -------
+            :class:`APyFloatArray`
+
+            Raises
+            -------
+            TypeError
+                If .....
+
+            Examples
+            -------
+
+            >>> from apytypes import APyFloatArray
+
+            Array `a`, array to summate across
+
+            >>> a = APyFloatArray(
+            ...     [1,2,3,4,5,6],
+            ...     int_bits=5,
+            ...     frac_bits=0
+            ... )
+
+            Array `b`, returned array where the specified axes are summated
+            >>> b = a.sum()
+            In this case `b` will be equal to `c`
+            >>> c = APyFloatArray(
+            ...     [21],
+            ...     int_bits=8,
+            ...     frac_bits=0
+            ... )
+
+            )pbdoc"
+        )
+
+        .def(
+            "cumsum",
+            &APyFloatArray::cumsum,
+            nb::arg("axis") = nb::none(),
+            R"pbdoc(
+            Returns the cumulative sum of the elements along a given axis.
+
+
+            Parameters
+            ----------
+            axis : int, optional
+                The axis to summate across. If not given an axis it will return the cumulative sum of the flattened array
+
+
+            Returns
+            -------
+            :class:`APyFloatArray`
+
+
```

Comment:
Some empty lines can be removed, but make sure to keep one empty line so that the formatting in the generated documentation is correct